PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -522,7 +605,8 @@ intptr_t CALLBACK Shortcut::run_dlgProc(UINT Message, WPARAM wParam, LPARAM lPar
 					{
 						if (LOWORD(wParam) == IDC_KEY_COMBO)
 						{
-							auto i = ::SendDlgItemMessage(_hSelf, LOWORD(wParam), CB_GETCURSEL, 0, 0);
+							auto iPullDown = ::SendDlgItemMessage(_hSelf, LOWORD(wParam), CB_GETCURSEL, 0, 0);
+							auto i = oemVirtualKeyUsed[iPullDown];
 							_keyCombo._key = namedKeyArray[i].id;

```

Comment:
Since we need the used OEM virtual key ID, we can do without namedKeyArray[i].id; in this PR.
Instead of storing the namedKeyArray index in oemVirtualKeyUsed, We store the ID directly (So in this case, the name **oemVkUsedIDs** is preferable).

I would suggest the following code
```cpp
	auto iSel = ::SendDlgItemMessage(_hSelf, LOWORD(wParam), CB_GETCURSEL, 0, 0);
	_keyCombo._key = oemVkUsedIDs[iSel];
```

instead of 
```cpp
	auto iPullDown = ::SendDlgItemMessage(_hSelf, LOWORD(wParam), CB_GETCURSEL, 0, 0);
	auto i = oemVirtualKeyUsed[iPullDown];
	_keyCombo._key = namedKeyArray[i].id;
```